_browser/blockstack_storage.md

```diff
@@ -220,48 +220,36 @@ These are the `UserSession.putFile`, `UserSession.getFile`, and `lookupProfile`
 
 1. Open the `src/components/Profile.js` file.
 
-2. Replace the initial state in the `constructor()` method so that it holds the key properties required by the app.
+2. Replace the initial state in the component method so that it holds the key properties required by the app.
 
-    This code constructs a Blockstack `Person` object to hold the profile. Your constructor should look like this:
+    This code constructs a Blockstack `Person` object to hold the profile. Your component should look like this:
 
     ```javascript
-    constructor(props) {
-      super(props);
-
-      this.state = {
-        person: {
-          name() {
-            return 'Anonymous';
-          },
-          avatarUrl() {
-            return avatarFallbackImage;
-          },
-        },
-        username: "",
-        newStatus: "",
-        statuses: [],
-        statusIndex: 0,
-        isLoading: false
-      };
+    export const Profile = ({ userData, handleSignOut }) => {
+      const [newStatus, setNewStatus] = React.useState('');
+      const [statuses, setStatuses] = React.useState([]);
+      const [statusIndex, setStatusIndex] = React.useState(0);
+      const [isLoading, setLoading] = React.useState(false);
+      const [username, setUsername] = React.useState(userData.username);
+      const [person, setPerson] = React.useState(new Person(userData.profile));
+      const { authOptions } = useConnect();
+      const { userSession } = authOptions;
+      // ...
     }
     ```
 
 
-3. Locate the `render()` method.
-4. Modify the `render()` method to add a text input and submit button to the 
+3. Modify the rendered result to add a text input and submit button to the 
    by replacing it with the code below:
 
     The following code renders the `person.name` and `person.avatarURL`
     properties from the profile on the display:
 
     ```javascript
-    render() {
-      const { handleSignOut, userSession } = this.props;
-      const { person } = this.state;
-      const { username } = this.state;
+    export const Profile = ({ userData, handleSignOut }) => {
+      // ... state setup from before
 
       return (
```

Comment:
This is quite confusing as well. We're implicitly assuming the reader will know that the `render` method in the file is to be replaced by a `return` statement. Not sure if everyone would get that right away.